crates/aptos/src/node/mod.rs

```diff
@@ -1526,6 +1532,49 @@ impl CliCommand<()> for BootstrapDbFromBackup {
     }
 }
 
+/// Checks the network connectivity of a node by dialing the node using noise
```

Comment:
This shows up in the help.  Please put

Checks the network connectivity of a node

Then on a different line describe what it uses and how it works.

When you type -h it gives the first line, --help gives the whole info

Reply:
Done, thanks! 😄